docs/sources/next/misc/usage-collection.md

```diff
@@ -6,7 +6,9 @@ weight: 02
 
 # Usage collection
 
-By default, k6 sends a usage report each time it is run, so that we can track how often people use it. This report can be turned off by setting the environment variable `K6_NO_USAGE_REPORT` or by adding the option `--no-usage-report` when executing k6.
+By default, k6 sends a usage report each time it is run, so that we can track relevant information to be able to build the product making better data-driven decisions. Prioritizing the features that benefit the most and reducing the impact of changes.
```

Comment:
```suggestion
By default, k6 sends an anonymous usage report each time it is run, so that we can track relevant information to be able to build the product making better data-driven decisions. Prioritizing the features that benefit the most and reducing the impact of changes.
```

I know that sometimes, in this kind of explanations, it is clarifying that the report is _anonymous_ or _ anonymized_, so may worth considering to mention it here as well, I guess.